spec/feature/account_home_spec.rb

```diff
@@ -0,0 +1,22 @@
+require("rails_helper")
+
+RSpec.describe("AccountHome", type: :feature) do
+  before do
+    stub_content_store_has_item("/")
+  end
+
+  context("/account/home") do
```

Comment:
It's strange that `GOVUK_PERSONALISATION_YOUR_ACCOUNT_URI` is not being modified here. 🤔

Reply:
Unless I've misunderstood something it doesn't really need to be in this test, because the default values for those methods are fine.